presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2723,6 +2731,37 @@ public void testGroupedExecution()
             assertQuery(colocatedOneGroupAtATime, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
             assertQuery(colocatedOneGroupAtATimeDynamic, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
 
+            //
+            // Window function
+            // ===============
+            ImmutableList.Builder<String> expectedCountValues = ImmutableList.builder();
+            expectedCountValues.addAll(nCopies(7333, "('O', 7333)"));
+            expectedCountValues.addAll(nCopies(7304, "('F', 7304)"));
+            expectedCountValues.addAll(nCopies(363, "('P', 363)"));
+            @Language("SQL") String expectedCountResult = format("VALUES %s", String.join(", ", expectedCountValues.build()));
+            assertQuery(
+                    colocatedOneGroupAtATime,
```

Comment:
I think we should rename this variable to some thing like `groupedExecutionOneGroupAtATime`. But we can do this in separate PR.